webapp/src/js/components/Plot.js

```diff
@@ -32,7 +30,24 @@ let Plot = createReactClass({
     };
   },
 
+  componentWillMount() {
+    Promise.all([
```

Comment:
You don't need `.all` here as you only have a single promise.

Reply:
Yup, it's just to demonstrate a mechanism for multiple dynamic imports. Hence the comment on the next line.

Reply:
I'll dynamically import `react-plotlyjs` as well, just for kicks.

Reply:
Ahh ok, makes sense